cluster/up.sh

```diff
@@ -18,7 +18,8 @@ set -ex
 
 source ./cluster/kubevirtci.sh
 CNAO_VERSIOV=0.35.0
-KUBEVIRT_VERSION=v0.29.0
+#use kubevirt latest release
+KUBEVIRT_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases/latest | grep .tag_name | sort -V | tail -1 | awk -F':' '{print $2}' | sed 's/,//' | xargs)
```

Comment:
We have to be very careful here since latest is not always latest, for example the latest could be a z-stream update and comming from release branch and that not the latest from master, so I propose that we mark the major.minor jere and take the latest z stream for example right now we have the following versions at kubevirt:
- v0.31.0-rc.1
- v0.30.3

the release branch KMP `release-v0.14.0` has to take `v0.30.3` but master has to take `v0.31.0-rc.1` so ideally what we would have at KUBEVIRT_VERSION is 0.30,0.31 etc...